src/main/kotlin/org/wycliffeassociates/otter/common/domain/content/RecordTake.kt

```diff
@@ -23,9 +23,9 @@ class RecordTake(
         private val waveFileCreator: IWaveFileCreator,
         private val launchPlugin: LaunchPlugin
 ) {
-    private fun getChunkCount(collection: Collection): Single<Int> = chunkRepository
+    private fun getVerseCount(collection: Collection): Single<Int> = chunkRepository
```

Comment:
is there a reason for changing vocabulary here?

Reply:
Yep, a chunk could now be a chapter or verse chunk. Here we only want to get verses, so `getVerseCount` is more accurate here